hugo/content/contribute.md

```diff
@@ -3,83 +3,279 @@ title: Contribute
 
-Mumble and all around it needs documentation. Great documentation for a great program.
+To find bugs and polish new releases we need testers.
 
-**Website**: This website has a public hugo source repository at [mumble-www](https://github.com/mumble-voip/mumble-www). You can report [issues](https://github.com/mumble-voip/mumble-www/issues), create [requests](https://github.com/mumble-voip/mumble-www/issues), and create pull requests for implemented change suggestions.
+If you want to test newest development versions, you have two options:
 
-**Wiki**: Our [documentation wiki](https://wiki.mumble.info) covers some more advanced topics than this website; for example for server administrators and contributors. Any contributions welcome, be it improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+- We provide development snapshots. You can download them from our [Downloads page]({{< relref "downloads" >}}).
+<!-- link to exact point? https://www.mumble.info/downloads/#development-snapshots -->
 
-**Wikipedia**: Someone that is not a development team member needs to maintain the [article at Wikipedia](https://en.wikipedia.org/wiki/Mumble_%28software%29). At the moment it’s missing references; try to find some. (This requires more work than just googling). Contributing to or creating a mumble article on non-English Wikipedia are also appreciated. See Promotion section below.
+- Build your own version of Mumble, see [Building Mumble](https://github.com/mumble-voip/mumble#building).
 
-**Ice methods**: We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+If you find any issues, take a look at [Report Bugs](#report-bugs-and-feature-requests).
 
-# Promotion
+<!-- mention documentation? Potential problem with TOC linking as the categories have the same name -->
+<!-- 
+### Documentation
+
+See [below](#documentation). -->
 
-Get the word out. This means more than writing about it on your blog ;)
+### Mumble theme
 
-* Ask (voice-)server hosters on if they also provide Mumble servers if you can’t find them.
-* Coordinate with one of the hosters providing free mumble servers and start promoting Mumble as the voicechat of choice for your favorite game.
-* Add mumble hosters you found to the {{< wiki Hosters />}} wiki page / update information.
-* Create How-Tos, Screencasts, Tutorials or Benchmarks on typical issues and topics, and publish them on prominent platforms.
-* Spread the word in groups – join our [Steam Group](https://steamcommunity.com/groups/Mumble_VoIP)
+<!-- Add more details, descriptions etc.? -->
+You find the Mumble theme in it's own [repository on Github](https://github.com/mumble-voip/mumble-theme).
 
-# Skinning
+#### Skins
```

Comment:
Isn't skinning and theming basically the same thing? :thinking:

Reply:
Kind of.
We could discuss a general renaming.

Reply:
I opened a discussion: https://github.com/mumble-voip/mumble/discussions/4647

Reply:
Renamed it, but not in the wiki.